WordPress/src/main/java/org/wordpress/android/ui/accounts/login/Login2FaFragment.java

```diff
@@ -54,13 +71,46 @@
     ArrayList<Integer> mOldSitesIDs;
 
     private String mEmailAddress;
+    private String mIdToken;
+    private String mNonce;
+    private String mNonceAuthenticator;
+    private String mNonceBackup;
+    private String mNonceSms;
     private String mPassword;
-
-    public static Login2FaFragment newInstance(String emailAddress, String password) {
+    private String mService;
+    private String mType;
+    private String mUserId;
+    private boolean isSocialLogin2fa;
+    private boolean isSocialLoginConnect;
```

Comment:
There are 100+ instances of `boolean [m](\w)+;` and 78 instances of `boolean [^m](\w)+;` variables in the WordPress codebase.  I don't think we've decided on anything.  We should clean those up when we implement code style automation.

Reply:
I think this PR is not the place to debate [codestyle guideline](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/CODESTYLE.md) changes but I see what you mean.

Let's keep it as is here (non-hungarian notation in the `isXXX` fields) but I'd highly suggest we stick to the current styleguide in subsequent PRs until it gets revised. Manually enforcing the styleguide in PR review (instead of say, using `checkstyle`) is already laborious so, let's not add debating on top of it 😸 .